sections/game_modes.tex

```diff
@@ -22,17 +22,25 @@ \subsection*{Cooperative}
 A Cooperative mode for 2--3 players where everyone shares the same goal.
 
 \subsection*{Tournament}
-A 1 vs.~1 mode with different way of building map and scoring. See the Tournament Book.
+A 1 vs.~1 mode with different way of building Map and scoring. See the Tournament Book.
 
-\subsection*{Battlefield expansion}
-The expansion adds two more 1 vs.~1 modes, without using Scenarios: Adventure, Skirmish. See the Battlefield expansion's rulebook.
+\vspace*{\fill}
+\begin{center}
+  \transparent{0.3}{\includegraphics[width=\linewidth, keepaspectratio]{\art/black_market.png}}
+\end{center}
 
-\subsubsection*{Adventure}
-The map exploration is replaced with Adventure cards. It ends with a battle on a big battlefield.
+\columnbreak
 
-\includegraphics[width=\linewidth]{\art/archer.jpg}
+The \textbf{Battlefield Expansion} adds two scenario-free 1 vs.~1 modes:
```

Comment:
I would agree with this rewording if the rewritten book provided the rule for these. But it doesn't yet

Reply:
Make it as you prefer, my main intention was to improve the layout.